woocommerce/woocommerce
# Merge WooCommerce Beta Tester Plugin

### All Submissions:






### Changes proposed in this Pull Request:



This pull request merges [`woocommerce/woocommerce-beta-tester`](https://github.com/woocommerce/woocommerce-beta-tester) and all of its history into `plugins/woocommerce-beta-tester`. This makes it appear as if the plugin was always developed within the monorepo.

### How to test the changes in this Pull Request:

1. Verify that the code is present in `plugins/woocommerce-beta-tester`2. 
2. Run `pnpm nx composer-install woocommerce-beta-tester`

From here you can mount it in whatever environment of your choosing. I would suggest _not_ trying to use it since it will squash your current woocommerce plugin! We're going to need to tweak it to use a different path to be compatible with the monorepo properly.

### Other information:




### Changelog entry

N/A

### FOR PR REVIEWER ONLY:
